### PR TITLE
Moved the setting into setup for Nvimtree

### PR DESCRIPTION
**Pull request by macarthy**
Options are currently being migrated into the setup function, you need to run require'nvim-tree'.setup() in your personal configurations. Setup should be run in a lua file or in a lua heredoc (:help lua-heredoc) if using in a vim file. Note that options under the g: command should be set BEFORE running the setup function.

I've moved the ones that have been migrated

### CODE REVIEW

**Comment by macarthy** [comment]
Just doing more testing and seem to have a issue with ``` { key = "g?", cb = tree_cb "toggle_help" }, ```  Can anyone confirm ?

**Comment by siduck76** [comment]
@macarthy the width is smaller than before 

image
 
 
 increasing it doesnt work either , can you confirm this?

**Comment by siduck76** [comment]
> Just doing more testing and seem to have a issue with `{ key = "g?", cb = tree_cb "toggle_help" },` Can anyone confirm ?

image

**Comment by siduck76** [comment]
nvim plugins gotta have stable branches imo :/

**Comment by macarthy** [comment]
I changed ``` g.nvim_tree_width = 25```

to 
```     view = {
       width = "25", } ```

**Review by siduck76 — APPROVED** [review]

**Comment by siduck76** [comment]
> I changed ` g.nvim_tree_width = 25`
> 
> to
> 
> ```
> view = {
>        width = "25", }
> ```

it should be only 25! since its a number ._.

File: lua/plugins/configs/nvimtree.lua [hunk 1]
```diff
@@ -102,4 +65,44 @@ require("nvim-tree").setup {
       enable = true,
       update_cwd = true,
    },
+   view = {
+      allow_resize = true,
+      side = "left",
+      width = 25,
+      mappings = {
+         list = {
+            { key = { "<CR>", "o", "<2-LeftMouse>" }, cb = tree_cb "edit" },
```
Comment (ashincoder) [inline lua/plugins/configs/nvimtree.lua:74]: I don't know why adding default keymappings.

Comment (siduck76) [reply lua/plugins/configs/nvimtree.lua:74]: @ashincoder we might change em in the future and it gets easier for the user to just look at the binds in the config if they dont know g? nvimtree keybind

**Comment by ashincoder** [comment]
I mean we don't to spoon feed the users anymore. 🤷.  Just send them
nvimtree.lua. Anyways no problem 😊

**Comment by siduck76** [comment]
@ashincoder agreed xd , ive seen we didnt make any changes in the default mappings of nvimtree so we'll remove it

**Comment by ashincoder** [comment]
Atleast you've agreed with me. 😆

**Pull request merged**